main/src/ca/uwaterloo/flix/language/errors/NameError.scala

```diff
@@ -38,12 +38,12 @@ object NameError {
     * @param loc2 the location of the use.
     */
   case class AmbiguousVarOrUse(name: String, loc: SourceLocation, loc1: SourceLocation, loc2: SourceLocation) extends NameError {
-    def summary: String = s"Ambiguous name. The name may refer to both a variable and a use."
+    def summary: String = s"Ambiguous name: '$name'. The name may refer to both a variable and a use of a name."
 
     def message(formatter: Formatter): String = {
       import formatter._
       s"""${line(kind, source.format)}
-         |>> Ambiguous name '${red(name)}'. The name may refer to both a variable and a use.
+         |>> Ambiguous name '${red(name)}'. The name may refer to both a variable and a use of a name.
```

Comment:
Would it be possible to show both occurrences?

Reply:
it does